src/app/static/src/js/components/runWorkflow/runWorkflowSummaryHeader.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+<header>
+    <h2 id="summary-header">Summary</h2>
+    <div id="summary-warning" class="row mt-3" v-if="hasMissingDependencies">
+        <div class="col-auto">
+            <alert-triangle-icon size="2x" stroke="red" class="custom-class"/>
+        </div>
+        <div class="col-auto">
+            <span class="d-inline-block pb-2">Some reports depend on the latest version of other reports that are not included in your workflow:</span>
+            <div class="missingDependency" v-for="(missing_dependencies, report) in workflowSummary.missing_dependencies" :key="report">
+                <span v-if="missing_dependencies.length" class="font-weight-bold"> {{ report }}</span>
+                <ul v-for="missing_dependency in missing_dependencies" class="styled"  :key="missing_dependency">
+                    <li>{{ missing_dependency }}</li>
+                </ul>
+            </div>
+        </div>
+    </div>
+</header>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import {WorkflowSummary} from "../../utils/types";
+import {AlertTriangleIcon} from "vue-feather-icons"
+
+interface Props {
+    workflowSummary: WorkflowSummary[] | null
+}
+
+interface Computed {
+    hasMissingDependencies: boolean;
+}
+export default Vue.extend<unknown, Computed, unknown, Props>({
+    name: "runWorkflowSummaryHeader",
+        components: {
+            AlertTriangleIcon
+        },
+    props: {
+        workflowSummary: {
+            type: Object || null,
+            required: true
+        }
+    },
+    computed: {
+        hasMissingDependencies() {
+            return this.workflowSummary && !!Object.keys(this.workflowSummary.missing_dependencies)
```

Comment:
Couldn't this just be 
`return this.workflowSummary && !!Object.keys(this.workflowSummary.missing_dependencies).length > 0` - we're just checking if there's anything in that object at all aren't we?

Reply:
I've had to change this back because even if there are no missing dependencies, the missing_dependencies property will still be populated by keys of empty arrays for each report, such as: 
![image](https://user-images.githubusercontent.com/57633087/153576524-a612470c-e896-47e4-9492-c91b21593890.png)

Reply:
Ah, I see..